includes/managers/image.php

```diff
@@ -82,7 +84,23 @@ public function get_details( $id, $size, $is_first_time ) {
 					],
 				];
 
-				$urls[ $size ] = Group_Control_Image_Size::get_attachment_image_src( $id, 'image', $instance );
+				$url = Group_Control_Image_Size::get_attachment_image_src( $id, 'image', $instance );
+
+				$thumbs_path = BFITHUMB_UPLOAD_DIR . '/' . basename( $url );
+
+				$image_meta = wp_get_attachment_metadata( $id );
+
+				$image_meta['sizes'][ 'elementor_' . $size ] = [
+					'file' => $thumbs_path,
+					'width' => $matches[1],
+					'height' => $matches[2],
+					'mime-type' => get_post_mime_type( $id ),
+				];
+
+				// Attach custom image to original.
```

Comment:
Redundant comment. If you really want, put that above the array assignment (Line 93)

Reply:
Done,